lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -818,6 +840,10 @@ def get_installed_snap_apps(self, name: str) -> List:
         """Query the snap server for apps belonging to a named, currently installed snap."""
         return self._request("GET", "apps", {"names": name, "select": "service"})
 
+    def put_snap_config(self, name: str, conf: Dict[str, Any]):
```

Comment:
Let's make this private `_put_snap_config`.

Reply:
Renamed, thanks!